code/integration/main/main.ino

```diff
@@ -56,33 +62,42 @@ void loop()
       String received_string = myWiFiService.Read();                    // get string from Wifly
       g_states = myParser.RunParser(received_string, 0, 0);             // interpret string
       String parser_return_string = myParser.Get_String_from_Parser();  // get string for factory
-      
-      switch ( g_states )                                               // evaluate next steps
+
+      switch ( g_states )
       {
-        case ERROR_STATE:
-          break;
-        case LOGIN_SUCCESSFUL:
-          break;
-        case LOGIN_PW_WRONG:
-          break;
-        case LOGOUT_SUCCESSFUL:
+        case ERROR_STATE:                           // error states have same result
+        case LOGIN_PW_WRONG:                        // error counter ++
+        case LOGOUT_PW_WRONG:                        
+        case ORDER_WRONG:                           
+        case ORDER_PW_WRONG:                        
+          g_error_count++;
+        case LOGIN_SUCCESSFUL:                      // client detected
```

Comment:
Schau bitte in Slack im Thread, wo du geschrieben hast: "irgend was beim eintragen ins Excel ist schief gegangen. Da ist ein Konflikt."